### PR TITLE
issue80-moved the table's data initiation into sql file

### LINKED ISSUE #80: Test database data loading control
## 🐛 Describe the bug

Loading test data to databse can't be configured externaly (load data or not)

## ⚠️ Current behavior

Loading test data in database controls from `DataLoader` code
```
@Override public void run(String... args) throws Exception {
    loadData();
  }
```
## ✅ Expected behavior

application.yml variable that controls the loaded data or not. May be create separate application.yml files for dev, prod and tests.

### PR DESCRIPTION
**Pull request by amirabramovich**
Bonus: added banner for application :)

## 📦 Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] My code follows the code style of this project(Do your best to follow code styles. If none apply just skip this).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by amirabramovich** [comment]
i see that the changes causes some tests to fail even though the application starts normally.
Got DataIntegrityViolationException, ill investigate it, you can check it too if you want since i won't be available in the next 3 days.

**Comment by DDmit04** [comment]
will it be possible to use current [postman configuration](https://github.com/jmprathab/MyHome/tree/master/postman) after this change? Environment varibles in this config are very comfortable for testing

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -25,7 +25,9 @@ server:
 # Spring
 ###############################################################################
 spring:
-  application.name: "myhome-service"
+  application:
+    name: "myhome-service"
+    version: 1.2.0-release
```
Comment (jmprathab) [inline src/main/resources/application.yml]: Let's not include this in the banner. As this gets outdated very soon, also we should take care of updating this after every release.
I am wondering if there is any way to automatically get it from Gradle configuration.

File: src/main/java/com/myhome/bootstrap/DataLoader.java [hunk 2]
```diff
@@ -59,7 +50,6 @@
 
   @Override public void run(String... args) throws Exception {
     //loadPostmanData();
```
Comment (jmprathab) [inline src/main/java/com/myhome/bootstrap/DataLoader.java:52]: Like @DDmit04 mentioned. We have to look for a way to include this in the `data.sql` file.

**Review by jmprathab — CHANGES_REQUESTED** [review]

**Comment by amirabramovich** [comment]
hi,
i fixed the banner, now it takes information from gradle. for some reason the application name & version are visable only by running with java -jar (please check the first answer [here](https://stackoverflow.com/questions/34519759/application-version-does-not-show-up-in-spring-boot-banner-txt)).
also i fixed the JUnits, issue was with the inserts order in data.sql , and also it seems that every integration test class loads data.sql before the tests so i temporarily merged the two test classes into single class (need to figure out later how to clean data before test class and implement it, maybe should create issue for it).
also regarding the postman changes, please create a separate issue for it and describe there exactly what is the purpose of postman initiation . 
regarding the issue which i took #80  - i pretty sure that task is completed here so we can merge and close the issue (and open few new tasks for the new issues).

File: src/main/java/com/myhome/bootstrap/DataLoader.java (hunk 2 above)
Comment (amirabramovich) [reply src/main/java/com/myhome/bootstrap/DataLoader.java:52]: please create new task for it

**Comment by mslowiak** [comment]
@amirabramovich `DirtyContext` annotation can help with that on this moment
https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/test/annotation/DirtiesContext.html

**Comment by amirabramovich** [comment]
@mslowiak
Tried that, didn't work for some reason :(

**Comment by jmprathab** [comment]
> @mslowiak
> Tried that, didn't work for some reason :(

Did you try it in conjunction with `@Transactional` ?

**Review by jmprathab — APPROVED** [review]
Looks good.

**Pull request merged**